src/mono/wasm/runtime-test.js

```diff
@@ -139,6 +139,13 @@ while (true) {
 }
 testArguments = args;
 
+function writeContentToFile(content, path)
```

Comment:
nit: use this function in the `if (enable_zoneinfo)` logic too, it has a duplicate of the code now.

Reply:
Good catch; will update.